src/bindings/python/wheel/fdupes_check.cmake

```diff
@@ -12,7 +12,13 @@ endforeach()
 
 find_program(fdupes_PROGRAM NAMES fdupes DOC "Path to fdupes")
 if(NOT fdupes_PROGRAM)
-    message(WARNING "Failed to find 'fdupes' tool, use 'sudo apt-get install fdupes' to install it")
+    set(fdupes_install_msg "refer to your platform's package manager or install it manually.")
+    if(Linux)
```

Comment:
`Linux` is not cmake variable. 

since `src/bindings/python/wheel/fdupes_check.cmake` is a separate file, I am not sure that `LINUX` (defined here https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/target_flags.cmake#L85-L87) is available in this file.

BTW, `Linux` is a target platform, while you can compile from macOS to Linux.
So, you need to make condition depending on host platform:

![{D534631C-3391-4B12-B092-65BDA3410F5B}](https://github.com/user-attachments/assets/ab3d7d65-53ef-4d05-a8f9-d7e059c5846f)

Reply:
looks like we need to refactor our project then

Reply:
why?